### PR TITLE
Add practical example for agent telemetry.

### PR DESCRIPTION
**Pull request by erikbaranowski**
Format .river files.

Clean up some documentation on the way.

### CODE REVIEW

File: modules/grafana-agent/telemetry-to-lgtm/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+| `prometheus_user`       | `string`   | The Prometheus remote write basic auth username. | | yes
+| `prometheus_password`   | `secret`   | The Prometheus remote write basic auth password. | | yes
+| `loki_endpoint`         | `string`   | Loki endpoint | | yes
+| `loki_user`             | `string`   | Loki basic auth username. | | yes
+| `loki_password`         | `secret`   | Loki basic auth password. | | yes
+| `tempo_endpoint`        | `string`   | Tempo Endpoint | | yes
+| `tempo_user`            | `string`   | Tempo basic auth username. | | yes
+| `tempo_password`        | `secret`   | Tempo basic auth password. | | yes
+
+Grafana Agent logs must be forwarded to the file at `grafana_agent_logpath`.
```
Comment (mattdurham) [inline modules/grafana-agent/telemetry-to-lgtm/README.md]: Thoughts on adding a quick snippet on how to do that?

Comment (erikbaranowski) [reply modules/grafana-agent/telemetry-to-lgtm/README.md]: good call, will do

**Comment by mattdurham** [comment]
Can we add a link to the main readme to the new module?

**Review by mattdurham — APPROVED** [review]
Two minor suggestions and then its good to go

**Pull request merged**